src/jbloader/loader/enable_ssh.c

```diff
@@ -21,14 +21,14 @@ void *enable_ssh(void *__unused _)
   char* dropbear_plist = NULL;
   switch (dyld_platform) {
     case PLATFORM_IOS:
-      dropbear_plist = "/cores/binpack/Library/LaunchDaemons/dropbear.plist";
+      dropbear_plist = "/cores/binpack/Library/LaunchDaemons/DropBear/dropbear.plist";
```

Comment:
is there a specific reason these were moved, or is it just to keep stuff organized?

Reply:
please be sure to put daemon plist files under `/cores/binpack/Library/LaunchDaemons/`.

Reply:
`load_daemon_from_path` will load all plists of a given path.
inorder to keep ths logic of `enable_ssh` and not arbitrary load all dropbear plists, i have moved them to separated folder.

@asdfugil